src/api/__mock__/cms/blocks.js

```diff
@@ -15,6 +15,15 @@ export default {
       url: '/foo',
     })),
   },
+  Promo: {
+    aligncontent: 'left',
+    name: 'Oldrin Maclorion',
+    avatar: 'https://picsum.photos/400/600',
```

Comment:
maybe we should consider a more generic name for the prop like "image" or "imageSrc".

We can think of blocks like reusable layout components that we may use in a different way later.